augur/export_v2.py

```diff
@@ -334,11 +334,17 @@ def set_author_on_nodes(node_metadata, raw_strain_info):
 
         node["author"] = {"author": author}
         if "title" in raw_strain_info[strain]:
-            node["author"]["title"] = raw_strain_info[strain]["title"].strip()
+            title = raw_strain_info[strain]["title"].strip()
+            if isValueValid(title):
+                node["author"]["title"] = title
         if "journal" in raw_strain_info[strain]:
-            node["author"]["journal"] = raw_strain_info[strain]["journal"].strip()
-        if "paper_url" in raw_strain_info[strain] and not raw_strain_info[strain]["paper_url"].strip("/").endswith("pubmed"):
-            node["author"]["paper_url"] = raw_strain_info[strain]["paper_url"].strip()
+            journal = raw_strain_info[strain]["journal"].strip()
+            if isValueValid(journal):
+                node["author"]["journal"] = journal
+        if "paper_url" in raw_strain_info[strain]:
+            paper_url = raw_strain_info[strain]["paper_url"].strip()
+            if isValueValid(paper_url) and not paper_url.strip("/").endswith("pubmed"):
```

Comment:
This feels like a pretty specific special-casing for bad data in our zika-tutorial / fauna.

I realize it existed prior to your change, so not a blocker for this PR, but thoughts on removing it and fixing the upstream data instead?

Reply:
I believe it will be super common to have fields in `metadata.tsv` that are `?` or `nan`, etc... for `paper_url`, `title`, etc... This will be the case for everyone. Not just our current Zika data. We don't want these included in the JSON, yes?

Reply:
Sorry, I wasn't clear. I didn't mean the `isValueValid()` test, but the `not paper_url.strip("/").endswith("pubmed")` test.

Reply:
(I saw that condition when skimming the diff, made a puzzled face, and went looking for what that's trying to catch since it's not obvious at all.)

Reply:
I see... not actually sure why this needs to be a condition. I'll investigate.

Reply:
I see where this is coming from now. I've fixed this issue in fauna:

https://github.com/nextstrain/fauna/commit/95c4cd9de1e87011ad7f7c2ea89c908aad2f278e

And removed this check on this `v6-author-export` branch.